### DISCUSSION TITLE
"Sending a new email using an alias is a premium feature." when using extension or mobile app.

### DISCUSSION THREAD

**Discussion by ramiro-zz** [simple-login/app#2156] [Self-hosting question]
Hi!

I've started using SimpleLogin self-hosted and have been getting good value out of it so far. However, I can't create a new reverse-alias when I'm using the Chrome extension or mobile app, it only works when I do it directly in the dashboard. I get the message:
"Sending a new email using an alias is a premium feature."
Is this seriously limited even for self-hosted users? Or is this a bug? Or am I doing something wrong?
I have set myself as a premium user in my self-hosted instance.

Thanks for any help!

**Comment by nguyenkims** [comment]
For self hosting, we recommend to set your account to lifetime so you don't run into any limits, more info on https://github.com/simple-login/app/blob/disable-pgp-onboarding-proton-mail/README.md#L534

**Reply by ramiro-zz** [reply]
hi, that link is a 404 but I assume you're referring to this step:

```
docker exec -it sl-db psql -U myuser simplelogin
UPDATE users SET lifetime = TRUE;
exit
```

I did that and I am configured as a lifetime premium member. As mentioned I can create reverse aliases on the web dashboard, just not the extension or the app.

**Comment by broodofqueen** [comment]
Same problem here, i am a lifetime premium member but still getting this "Sending a new email using an alias is a premium feature." error.

Thanks end regards for any help.

**Reply by ramiro-zz** [reply]
I have not had any luck with this still, but it does randomly work. For example right now when I tested, the Android app worked but the chrome extension gave the error.
